server/game/cards/02_SHD/events/SmugglersAid.ts

```diff
@@ -0,0 +1,25 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { CardType, RelativePlayer } from '../../../core/Constants';
+
+export default class SmugglersAid extends EventCard {
+    protected override getImplementationId () {
+        return {
+            id: '0866321455',
+            internalName: 'smugglers-aid',
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.setEventAbility({
+            title: 'Heal 3 damage from you base',
+            targetResolver: {
```

Comment:
Similar situation here as for Make an Opening, except that this case is much simpler since there is only one effect. Because we know the target in advance (controller's base), we do not need a target resolve. Can just do the `immediateEffect` directly:

```typescript
public override setupCardAbilities() {
    this.setEventAbility({
        title: 'Heal 3 damage from your base',
        immediateEffect: AbilityHelper.immediateEffects.heal((context) => ({ target: context.source.controller.base, amount: 3 }))
    });
}
```

It might be a little confusing that we pass in the `context` to the `heal()`, but everything that is under `immediateEffects` has the option to accept either (a) a properties object or (b) a function that accepts `context` and generates a properties object. In this case we do option (b) because we need to get the source card so that we can find out who is the controlling player whose base will be healed.